core/src/main/scala/org/ensime/util/DiffUtil.scala

```diff
@@ -23,8 +25,39 @@ object DiffUtil {
     if (file.exists)
       format.format(new Date(file.lastModified()))
     else {
-      format.setTimeZone(TimeZone.getTimeZone("UTC"))
-      format.format(new Date(0L))
+      epoch
     }
   }
+
+  private def epoch: String = {
+    val sdf = new SimpleDateFormat("yyyy-MM-dd hh:mm:ss Z")
+    sdf.setTimeZone(TimeZone.getTimeZone("UTC"))
+    sdf.format(new Date(0L))
+  }
+
+  def newFileDiff(text: Seq[String], file: File): String = {
+    val diff = StringBuilder.newBuilder
+    val originalInfo = s"${file.getAbsolutePath} $epoch"
+    val revisedInfo = s"${file.getAbsolutePath} ${fileModificationTimeOrEpoch(file)}"
+
+    diff ++= s"--- $originalInfo$lineSep"
```

Comment:
actually, doing this by hand seems like we're duplicating tested code from `DiffUtils`. Does it not have a way of generating a fresh file?

Imagine all the corner cases that we're not considering, like diffs of `diff` files and so on.